src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: var(--card-width);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: var(--margin);
+  transition-duration: 300ms;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(120%);
+  }
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__discription {
```

Comment:
There's a typo in the class name. It should be '&__description' not '&__discription'.